oap-logstream/src/main/java/oap/logstream/disk/DefaultWriter.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) Open Application Platform Authors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package oap.logstream.disk;
+
+import com.google.common.io.CountingOutputStream;
+import lombok.extern.slf4j.Slf4j;
+import oap.io.IoStreams;
+import oap.io.IoStreams.Encoding;
+import oap.logstream.LogId;
+import oap.logstream.LoggerException;
+import oap.logstream.Timestamp;
+import oap.template.BinaryInputStream;
+import oap.template.TemplateAccumulatorString;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.function.Consumer;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+@Slf4j
+public class DefaultWriter extends AbstractWriter<CountingOutputStream> {
```

Comment:
Does `Default` means TSV writer? If yes, it should be renamed to TSV and names as default in docs and configs. Sometime it can become not a default one

Reply:
I can see it renamed in a new commit. ok